src/main/java/com/google/ehub/data/CommentDataManager.java

```diff
@@ -0,0 +1,38 @@
+package com.google.ehub.data;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import java.util.*;
+
+public class CommentDataManager{
+
+  public void addItemComment(String message, long timestamp){
+    Entity commentEntity = new Entity("ItemPageComments");
+    commentEntity.setProperty("message", message);
```

Comment:
I suggest creating keys for all of the string values such as "ItemPageComments" and "message". 
```
private static final MESSAGE_PROPERTY_KEY = "message";
private static final TIMESTAMP_PROPERTY_KEY = "timestamp";
```

Reply:
Noted.

Reply:
Agreed with Bryan's suggestion. Its better to store these hardcoded strings as constants, especially when they are used in multiple places. In the future, if you need to change the string, you only need to update it in one place. This also prevents accidental typos.